.github/workflows/pr-requirements-check.yml

```
@@ -0,0 +1,101 @@
+# From: https://levelup.gitconnected.com/enforcing-jira-ticket-formatting-in-github-pr-titles-and-commit-messages-78d9755568df
+name: "PR Requirements Check"
+permissions:
+  contents: read
+  pull-requests: write
+on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)